src/R/Components/Impl/ConnectionManager/Implementation/ViewModel/ConnectionViewModel.cs

```diff
@@ -100,6 +101,29 @@ internal sealed class ConnectionViewModel : BindableBase, IConnectionViewModel {
             set { SetProperty(ref _isConnected, value); }
         }
 
+        /// <summary>
+        /// Tooltip when hovered over connection name
+        /// </summary>
+        public string Information {
```

Comment:
Also, please add it to the interface

---
In reply to: [77930046](https://github.com/Microsoft/RTVS/pull/2267#discussion_r77930046) [](ancestors = 77930046)